cxplat/inc/winuser/cxplat_winuser.h

```diff
@@ -24,3 +25,10 @@ typedef struct _cxplat_rundown_reference
 {
     void* reserved;
 } cxplat_rundown_reference_t;
+
+typedef struct _cxplat_lookaside_list
```

Comment:
Another wider proposal, not to be made in this PR: Instead of defining structs `typedef struct _<X> { ... } <X>_t` can we use `<X>_t` for both parts?

Reply:
`struct _<X>` is needed in cases where there are forward references, so it's often common practice to always add it in case forward references are needed later, and for consistency. but it's also ok though to only do it when a forward reference is actually done.

Reply:
I'm not sure I understand. MsQuic follows the design I described above. When we need to declare (before use) the type, we simply do:
```c
typedef struct QUIC_BINDING QUIC_BINDING;
```

Reply:
Got it, I misunderstood your comment.  Sounds fine to me.